src/lib/convenience/RegisterProviderRequest.h

```diff
@@ -50,7 +50,7 @@ typedef struct RegisterProviderRequest
   RegisterProviderRequest();
 
   std::string  render(Format format, std::string indent);
-  std::string  check(RequestType requestType, Format format, std::string indent, std::string preError, int counter);
+  std::string  check(ConnectionInfo* ciP, RequestType requestType, Format format, std::string indent, std::string preError, int counter);
```

Comment:
Ok not closing it, but al least to mention this PR in that issue comments as an step done towards that objective.